src/gui/search/searchjobwidget.cpp

```diff
@@ -208,7 +217,15 @@ void SearchJobWidget::cancelSearch()
 
 void SearchJobWidget::downloadTorrents(const AddTorrentOption option)
 {
-    const QModelIndexList rows {m_ui->resultsBrowser->selectionModel()->selectedRows()};
+    if (!BitTorrent::Session::instance()->isRestored())
+    {
+        QMessageBox::warning(this, tr("BitTorrent Session isn't restored")
```

Comment:
The title should say the current action (Downloading torrents) failed.

This applies to other similar messages.